android/modules/ui/src/java/ti/modules/titanium/ui/android/AndroidModule.java

```diff
@@ -81,6 +82,34 @@
 	@Kroll.constant public static final int WEBVIEW_LOAD_CACHE_ONLY = WebSettings.LOAD_CACHE_ONLY;
 	@Kroll.constant public static final int WEBVIEW_LOAD_CACHE_ELSE_NETWORK = WebSettings.LOAD_CACHE_ELSE_NETWORK;
 
+	@Kroll.constant public static final int GRAVITY_AXIS_CLIP = Gravity.AXIS_CLIP;
+	@Kroll.constant public static final int GRAVITY_AXIS_PULL_AFTER = Gravity.AXIS_PULL_AFTER;
+	@Kroll.constant public static final int GRAVITY_AXIS_PULL_BEFORE = Gravity.AXIS_PULL_BEFORE;
+	@Kroll.constant public static final int GRAVITY_AXIS_SPECIFIED = Gravity.AXIS_SPECIFIED;
+	@Kroll.constant public static final int GRAVITY_AXIS_X_SHIFT = Gravity.AXIS_X_SHIFT;
+	@Kroll.constant public static final int GRAVITY_AXIS_Y_SHIFT = Gravity.AXIS_Y_SHIFT;
+	@Kroll.constant public static final int GRAVITY_BOTTOM = Gravity.BOTTOM;
+	@Kroll.constant public static final int GRAVITY_CENTER = Gravity.CENTER;
+	@Kroll.constant public static final int GRAVITY_CENTER_HORIZONTAL = Gravity.CENTER_HORIZONTAL;
+	@Kroll.constant public static final int GRAVITY_CENTER_VERTICAL = Gravity.CENTER_VERTICAL;
+	@Kroll.constant public static final int GRAVITY_CLIP_HORIZONTAL = Gravity.CLIP_HORIZONTAL;
+	@Kroll.constant public static final int GRAVITY_CLIP_VERTICAL = Gravity.CLIP_VERTICAL;
+	@Kroll.constant public static final int GRAVITY_DISPLAY_CLIP_HORIZONTAL = Gravity.DISPLAY_CLIP_HORIZONTAL;
+	@Kroll.constant public static final int GRAVITY_DISPLAY_CLIP_VERTICAL = Gravity.DISPLAY_CLIP_VERTICAL;
+	@Kroll.constant public static final int GRAVITY_END = Gravity.END;
+	@Kroll.constant public static final int GRAVITY_FILL = Gravity.FILL;
+	@Kroll.constant public static final int GRAVITY_FILL_HORIZONTAL = Gravity.FILL_HORIZONTAL;
+	@Kroll.constant public static final int GRAVITY_FILL_VERTICAL = Gravity.FILL_VERTICAL;
+	@Kroll.constant public static final int GRAVITY_HORIZONTAL_GRAVITY_MASK = Gravity.HORIZONTAL_GRAVITY_MASK;
+	@Kroll.constant public static final int GRAVITY_LEFT = Gravity.LEFT;
+	@Kroll.constant public static final int GRAVITY_NO_GRAVITY = Gravity.NO_GRAVITY;
+	@Kroll.constant public static final int GRAVITY_RELATIVE_HORIZONTAL_GRAVITY_MASK = Gravity.RELATIVE_HORIZONTAL_GRAVITY_MASK;
+	@Kroll.constant public static final int GRAVITY_RELATIVE_LAYOUT_DIRECTION = Gravity.RELATIVE_LAYOUT_DIRECTION;
+	@Kroll.constant public static final int GRAVITY_RIGHT = Gravity.RIGHT;
+	@Kroll.constant public static final int GRAVITY_START = Gravity.START;
+	@Kroll.constant public static final int GRAVITY_TOP = Gravity.TOP;
+	@Kroll.constant public static final int GRAVITY_VERTICAL_GRAVITY_MASK = Gravity.VERTICAL_GRAVITY_MASK;
```

Comment:
4 of those were added in API Lvl 14, I'll add another commit in a sec with a compatibility shim for them. All `Ti.UI.Android.GRAVITY_*` constants will work on every Android version then.
In this case, do I need to document the description param?